<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ItemEditorDialog</class>
 <widget class="QDialog" name="ItemEditorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>407</width>
    <height>414</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChangeMe!</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="9" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <spacer>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>131</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="edtCode">
     <property name="maxLength">
      <number>8</number>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLineEdit" name="edtName">
     <property name="maxLength">
      <number>64</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblName">
     <property name="text">
      <string>&amp;Наименование</string>
     </property>
     <property name="buddy">
      <cstring>edtName</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lblCode">
     <property name="text">
      <string>&amp;Код</string>
     </property>
     <property name="buddy">
      <cstring>edtCode</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblProg">
     <property name="text">
      <string>&amp;Подпрограмма</string>
     </property>
     <property name="buddy">
      <cstring>edtProg</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QLineEdit" name="edtProg">
     <property name="maxLength">
      <number>128</number>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QLineEdit" name="edtPreferentArchiver">
     <property name="maxLength">
      <number>128</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lblPreferentArchiver">
     <property name="text">
      <string>&amp;Архиватор</string>
     </property>
     <property name="buddy">
      <cstring>edtPreferentArchiver</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QCheckBox" name="chkEmailRequired">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lblEmailRequired">
     <property name="text">
      <string>&amp;Отправлять по эл.почте</string>
     </property>
     <property name="buddy">
      <cstring>chkEmailRequired</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="lblEmailTo">
     <property name="text">
      <string>Aдрес &amp;эл.почты</string>
     </property>
     <property name="buddy">
      <cstring>edtEmailTo</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="2">
    <widget class="QLineEdit" name="edtEmailTo">
     <property name="maxLength">
      <number>64</number>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="lblSubject">
     <property name="toolTip">
      <string>используйте {Name}
для подстановки
значения параметра
Name</string>
     </property>
     <property name="text">
      <string>&amp;Тема сообщения</string>
     </property>
     <property name="buddy">
      <cstring>edtSubject</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="2">
    <widget class="QLineEdit" name="edtSubject">
     <property name="maxLength">
      <number>128</number>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="lblMessage">
     <property name="toolTip">
      <string>используйте {Name}
для подстановки
значения параметра
Name</string>
     </property>
     <property name="text">
      <string>&amp;Заготовка сообщения</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="buddy">
      <cstring>edtMessage</cstring>
     </property>
    </widget>
   </item>
   <item row="7" column="1" rowspan="2" colspan="2">
    <widget class="QTextEdit" name="edtMessage"/>
   </item>
   <item row="8" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>edtCode</tabstop>
  <tabstop>edtName</tabstop>
  <tabstop>edtProg</tabstop>
  <tabstop>edtPreferentArchiver</tabstop>
  <tabstop>chkEmailRequired</tabstop>
  <tabstop>edtEmailTo</tabstop>
  <tabstop>edtSubject</tabstop>
  <tabstop>edtMessage</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ItemEditorDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>162</x>
     <y>406</y>
    </hint>
    <hint type="destinationlabel">
     <x>121</x>
     <y>284</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ItemEditorDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>242</x>
     <y>406</y>
    </hint>
    <hint type="destinationlabel">
     <x>293</x>
     <y>284</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
